antarest/launcher/adapters/slurm_launcher/slurm_launcher.py

```diff
@@ -444,7 +449,6 @@ def _handle_success(self, study: StudyDTO) -> None:
                 study.name, JobStatus.FAILED, msg, None
             )
             logger.error(msg, exc_info=e)
-            raise
```

Comment:
Je me souviens avoir ajouté un `raise` explicitement pour voir les erreurs et les remonter au niveau de l’appelant. Si le on supprime, cela signifie que l’erreur est masquée.

Je suis d’accord pour supprimer le `raise` uniquement si j’ai une analyse claire des conséquences. À la suite de cela, il convient de documenter le code à l’aide d’un commentaire explicatif précis sans quoi, lors de la prochaine revue de code, je rajouterais probablement le `raise` 😜

Reply:
The "catch all" in the loop itself avoids to kill the slurm loop, which caused the progress of computations to not be monitored anymore (we had to restart the app to fix it).

However it's not enough to add that:
the individual result handling here must not raise, or else we will skip the handling of next results, and also the "cleaning" part.

To be honest we should have a unit test to check that behaviour, this really was a quick fix for deployment.

Where do you prefer to add comments? on this line, or on docstring?